test/domains/vass/domain-and-subdomain-args.compopts

```diff
@@ -0,0 +1,2 @@
+# the test relies on --checks
```

Comment:
This is not test/domains/vass/domain-arg-query-expr.chpl, which is what you said you ran :)

Reply:
D'oh. This is the correct file to edit. I have verified that this file passes with the compopts and updated the OP